docs/srcs/comparison.md

```diff
@@ -1132,4 +1153,4 @@ There are many mod managers around these days and it's may be a bit difficult to
 * [AL](https://www.nexusmods.com/users/6409802) (MO2 Dev)
 * [Yggdrasil75](https://www.nexusmods.com/users/25321034) (Vortex virtuoso)
 * [Sharlikran](https://www.nexusmods.com/users/25321034) (Wrye Bash contributor)
-* We found no one for NMM yet :/
+* NMM updates by Sharlikran
```

Comment:
Going to redo the credits list to be per tool, because as time goes on, multiple people may contribute to the the same tool, or one person to multiple tools, as Sharlikran did.